compute_tools/src/compute.rs

```diff
@@ -1826,8 +1828,6 @@ impl ComputeNode {
             })?;
         }
 
-        self.pg_reload_conf()?;
```

Comment:
let's keep this one? 

If line 1813 is executed, then this will apply max cluster size again.

Reply:
+1

Reply:
fixed, thanks :)